tests/tests_suite_test.go

```diff
@@ -357,6 +368,31 @@ func (s *existingSspStrategy) SkipIfUpgradeLane() {
 	skipIfUpgradeLane()
 }
 
+func (s *existingSspStrategy) waitForDataImportCrons() {
+	Eventually(func(g Gomega) {
+		for _, dataImportCronTemplate := range s.ssp.Spec.CommonTemplates.DataImportCronTemplates {
+			importCron := &cdiv1beta1.DataImportCron{}
+			importCronNamespacedName := types.NamespacedName{
+				Namespace: dataImportCronTemplate.Namespace,
+				Name:      dataImportCronTemplate.Name,
+			}
+			if importCronNamespacedName.Namespace == "" {
+				importCronNamespacedName.Namespace = internal.GoldenImagesNamespace
+			}
+
+			g.Expect(apiClient.Get(ctx, importCronNamespacedName, importCron)).To(Succeed(),
+				fmt.Sprintf("Could not get DataImportCron: %s", importCronNamespacedName.String()))
```

Comment:
nit: Expect takes format strings without `fmt.Sprintf`

Reply:
Thanks, I didn't know about that.